pom.xml

```diff
@@ -35,13 +35,15 @@ THE SOFTWARE.
     <groupId>io.jenkins.plugins</groupId>
```

Comment:
Parent POM could be updated to 4.4

Reply:
So I looked into it, seems it wasn't `jenkins.version` whose support was dropped, but rather `jenkins-core.version`. Ill allow dependabot to go ahead and do this.